recipes/smallgenomeutilities/0.2/meta.yaml

```diff
@@ -0,0 +1,67 @@
+{% set name = "smallgenomeutilities" %}
+{% set version = "0.2" %}
+{% set md5 = "f8b222892e9f18bb135efd5802a878a4" %}
+{% set sha256 = "802ca5f85a14e8243aa44e5f4fb771691f0c80689cf3ecb1edce5382ce0ecda3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name|lower }}-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/f8/e2/54b075294a3247595abf9f7d7314eddc04c77a049398157514b65087fe6a/{{ name|lower }}-{{ version }}.tar.gz
```

Comment:
You should rather use this url `https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz` as yours does not work with 0.1

Reply:
Please do this, hardcoding hashes is a no-go